test/powershell/Modules/Microsoft.PowerShell.Management/Get-Process.Tests.ps1

```diff
@@ -15,3 +15,21 @@ Describe "Get-Process" -Tags "CI" {
         Get-Process | foreach-object { $_.Name | Should Not BeNullOrEmpty }
     }
 }
+
+Describe "Get-Process Formatting" -Tags "Feature" {
+    It "Should not have Handle in table format header" {
+        $types = "System.Diagnostics.Process","System.Diagnostics.Process#IncludeUserName"
+
+        foreach ($type in $types)
+        {
+            $formatData = Get-FormatData -TypeName $type -PowerShellVersion $PSVersionTable.PSVersion
+            $tableControls = $formatData.FormatViewDefinition | Where-Object {$_.Control -is "System.Management.Automation.TableControl"}
+            foreach ($tableControl in $tableControls)
+            {
+                $tableControl.Control.Headers.Label -match "Handle*" | Should BeNullOrEmpty
+                # verify that rows without headers isn't the handlecount (as PowerShell will create a header that matches the property name)
+                $tableControl.Control.Rows.Columns.DisplayEntry.Value -eq "HandleCount" | Should BeNullOrEmpty
+            }
+        }
+    }
+}
```

Comment:
I've read the arguments for requiring newlines at the end of the file and don't feel it's worth calling out in reviewing code.

If it really is useful, we can write a tool to fix the files and run the tool on a semi-regular basis.